### PR TITLE
Sentinel exporter env

### LINKED ISSUE #460: Sentinel exporter metrics are not scraped
### Expected behaviour

What do you want to achieve?
Sentinel exporter metrics should be scraped when we set the below in CRD.

exporter:
   enabled: true

### Actual behaviour

What is happening? Are all the pieces created? Can you access to the service?

Since redisexporter has the below default values, the sentinel exporter is trying to connect to 6379 port instead of 26379 and it is listening to redis://localhost:6379 instead of redis://localhost:26379

Ref: https://github.com/oliver006/redis_exporter

web.listen-address	REDIS_EXPORTER_WEB_LISTEN_ADDRESS	Address to listen on for web interface and telemetry, defaults to 0.0.0.0:9121.

redis.addr	REDIS_ADDR	Address of the Redis instance, defaults to redis://localhost:6379.

After setting the above env variables, it is working as expected. This can be passed as env variables for sentinel exporter itself to  avoid setting it in CRD.

### PR DESCRIPTION
**Pull request by samof76**
Fixes #460  .

Changes proposed on the PR:
- Set the following env on the exporter
  - REDIS_ADDR
  - REDIS_EXPORTER_WEB_LISTEN_ADDRESS
  - REDIS_ALIAS(may be not required, but useful)

### CODE REVIEW

File: operator/redisfailover/service/generator.go [hunk 1]
```diff
@@ -680,6 +694,11 @@ func createSentinelExporterContainer(rf *redisfailoverv1.RedisFailover) corev1.C
 		},
 		Resources: resources,
 	}
+
+	if rf.Spec.Sentinel.Exporter.Env != nil {
```
Comment (ese) [inline operator/redisfailover/service/generator.go:698]: Why is this needed? I think its already done before, in the line 673

Comment (samof76) [reply operator/redisfailover/service/generator.go]: Oh yes. removed!

**Review by ese — APPROVED** [review]
